src/main/java/com/sublinks/sublinksapi/api/lemmy/v3/comment/controllers/CommentController.java

```diff
@@ -520,7 +520,8 @@ GetCommentsResponse list(@Valid final GetComments getCommentsForm, final JwtPers
     Optional<Person> person = getOptionalPerson(principal);
     aclService.canPerson(person.orElse(null))
         .performTheAction(RolePermissionCommentTypes.READ_COMMENT)
-        .orElseThrow(() -> new ResponseStatusException(HttpStatus.UNAUTHORIZED, "unauthorized"));
+        .orElseThrow(() -> new ResponseStatusException(HttpStatus.UNAUTHORIZED,
+            "unauthorized_to_read_comments"));
```

Comment:
The security person in me prefers the original (less information returned). Do we really need to explain _what_ the person was unauthorized to do? Or just say "unauthorized" is enough? (same goes for later in this PR where you return a specific authorization error)

Reply:
A frontend would really like this as it would be able to allow translation of specific exceptions and not just generic unauthorized

Reply:
I can understand that, but from a security standpoint you are sharing more information than you need to. Saying "you aren't authorized to view these comments" now tells an attacker that they have an account that has SOME permissions, they just need to explore more. Saying "not authorized" doesn't tell the attacker whether the credentials are good or if they just don't have the right permissions.

Reply:
Ok will change that. ( Everywhere else i will change when i see it )